dev-build/app/Global.scala

```diff
@@ -69,6 +69,7 @@ with ConfigAgentLifecycle
 with SurgingContentAgentLifecycle
 with SectionsLookUpLifecycle
 with ABTHeadlinesLifecycle
+with MostPopularFacebookAutoRefresh.Lifecycle
```

Comment:
I don't mind the naming style here, but it should be consistent. Otherwise each `with` depends on developer preference, and becomes hard to read.